### PR TITLE
ocamlPackages-wasm: 1.0 -> 1.1

### PR DESCRIPTION
**Pull request by ggreif**
###### Motivation for this change

See https://opam.ocaml.org/packages/wasm/wasm.1.1

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ggreif** [comment]
@GrahamcOfBorg build ocamlPackages.wasm ocamlPackages_latest.wasm

File: pkgs/development/ocaml-modules/wasm/default.nix [hunk 1]
```diff
@@ -1,19 +1,19 @@
 { stdenv, fetchFromGitHub, ocaml, findlib, ocamlbuild }:
 
 if !stdenv.lib.versionAtLeast ocaml.version "4.02"
- || stdenv.lib.versionAtLeast ocaml.version "4.08"
+ || stdenv.lib.versionAtLeast ocaml.version "4.10"
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/wasm/default.nix:4]: iMHO this line should be removed.

**Comment by nomeata** [comment]
I believe `wasm-1.1.1` was submitted to opam today (see this tag); maybe we can jump directly to that?

**Comment by ggreif** [comment]
> I believe `wasm-1.1.1` was submitted to opam today (see this tag); maybe we can jump directly to that?

I only see `1.1 (latest)` in the popup. Can you give an URL?

**Comment by vbgl** [comment]
https://github.com/WebAssembly/spec/releases/tag/opam-1.1.1

**Comment by nomeata** [comment]
Seems it hasm't reached opam yet. No idea what the process is or how to get insight into it. Is there a bugtracker?

File: pkgs/development/ocaml-modules/wasm/default.nix (hunk 1 above)
Comment (ggreif) [reply pkgs/development/ocaml-modules/wasm/default.nix]: Done.

**Comment by nomeata** [comment]
https://opam.ocaml.org/packages/wasm/ has 1.1.1 now

**Comment by vbgl** [comment]
@GrahamcOfBorg build wasm ocaml-ng.ocamlPackages_4_10.wasm

**Comment by vbgl** [comment]
This might be worth porting to 20.03.

**Comment by ggreif** [comment]
See #87476 for `opam-1.1.1`.

**Pull request merged**